packages/filebrowser/src/browser.ts

```diff
@@ -404,6 +420,7 @@ export class FileBrowser extends SidePanel {
   private _showLastModifiedColumn: boolean = true;
   private _useFuzzyFilter: boolean = true;
   private _showHiddenFiles: boolean = false;
+  private _showFileCheckboxes: boolean = true;
```

Comment:
Similar to above: should this be opt-in or opt-out?

```suggestion
  private _showFileCheckboxes: boolean = false;
```